com.unity.render-pipelines.universal/Tests/Editor/Unity.RenderPipelines.Universal.Editor.Tests.asmdef

```diff
@@ -1,9 +1,11 @@
 {
     "name": "Unity.RenderPipelines.Universal.Editor.Tests",
+    "rootNamespace": "",
     "references": [
         "GUID:15fc0a57446b3144c949da3e2b9737a9",
         "GUID:c579267770062bf448e75eb160330b7f",
         "GUID:df380645f10b7bc4b97d4f5eb6303d95",
+        "GUID:3eae0364be2026648bf74846acb8a731",
```

Comment:
which assembly reference is being added?

Reply:
Removed in b139c5ffb592c82cc87c38a42b622816ade62209.